usr/share/rear/output/RAWDISK/Linux-i386/260_create_syslinux_efi_bootloader.sh

```diff
@@ -35,7 +35,7 @@ local efi_boot_directory="$RAWDISK_BOOT_EFI_STAGING_ROOT/BOOT"
 
 mkdir $v -p "$efi_boot_directory" || Error "Could not create $efi_boot_directory"
 
-cp $v "$syslinux_efi" "$efi_boot_directory/BOOTX64.EFI" >&2
+cp $v "$syslinux_efi" "$efi_boot_directory/BOOT${EFI_ARCH_UPPER}.EFI" >&2
```

Comment:
Same as
https://github.com/rear/rear/pull/3157/files#r1511412487

and `>&2` should be usually no longer needed
since stdout and stderr are handled same, cf.
https://github.com/rear/rear/wiki/Coding-Style#what-to-do-with-stdin-stdout-and-stderr